grpc/proto/search_get_v1.proto

```diff
@@ -0,0 +1,267 @@
+syntax = "proto3";
+
+package weaviategrpc;
+
+import "base.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "github.com/weaviate/weaviate/grpc/generated;protocol";
+option java_package = "io.weaviate.grpc.protocol";
+option java_outer_classname = "WeaviateProtoSearchGet";
+
+message SearchRequestV1 {
+  //required
+  string collection = 1;
+
+  // parameters
+  string tenant = 10;
+  optional ConsistencyLevel consistency_level = 11;
+
+  // what is returned
+  optional PropertiesRequest properties = 20;
+  optional MetadataRequest metadata = 21;
+  optional GroupBy group_by = 22;
+
+  // affects order and length of results. 0/empty (default value) means disabled
+  uint32 limit = 30;
+  uint32 offset = 31;
+  uint32 autocut = 32;
+  string after = 33;
+  // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
+  repeated SortBy sort_by = 34;
+
+  // matches/searches for objects
+  optional Filters filters = 40;
+  optional Hybrid hybrid_search = 41;
+  optional BM25 bm25_search = 42;
+  optional NearVector near_vector = 43;
+  optional NearObject near_object = 44;
+  optional NearTextSearch near_text = 45;
+  optional NearImageSearch near_image = 46;
+  optional NearAudioSearch near_audio = 47;
+  optional NearVideoSearch near_video = 48;
+
+  optional GenerativeSearch generative = 60;
```

Comment:
the last number is `48` so generative should be `50` not `60`

Reply:
There are 3 more coming (near_depth/ITU/somethingIForgot), so we will be over 50 here.